### PR TITLE
Remote grouping

### PR DESCRIPTION
**Pull request by AlekseyMartynov**

### CODE REVIEW

File: net/DevExtreme.AspNet.Data/RemoteGrouping/RemoteGroupExpressionCompiler.cs [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+                    Expression.New(_remoteGroupClassType.GetConstructor(Type.EmptyTypes)),
+                    projectionBindings
+                ),
+                param
+            );
+
+            return Expression.Call(typeof(Queryable), "Select", new[] { param.Type, _remoteGroupClassType }, target, projectionLambda);
+        }
+
+        void AddAggregateBindinge(ICollection<MemberAssignment> bindingList, Expression aggregateTarget, IList<Expression> selectorExprList, IList<ParameterExpression> summaryParams, IList<string> summaryTypes, string bindingFieldPrefix) {
```
Comment (IlyaKhD) [inline net/DevExtreme.AspNet.Data/RemoteGrouping/RemoteGroupExpressionCompiler.cs]: Shouldn't it be `AddAggregateBinding` ? without trailing 'e' ?

Comment (AlekseyMartynov) [inline net/DevExtreme.AspNet.Data/RemoteGrouping/RemoteGroupExpressionCompiler.cs]: Thanks. Will fix.

File: net/DevExtreme.AspNet.Data/RemoteGrouping/RemoteGroupExpressionCompiler.cs [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace DevExtreme.AspNet.Data.RemoteGrouping {
+
+    class RemoteGroupExpressionCompiler<T> : ExpressionCompiler {
+        const int MAX_GROUPS = 8;
```
Comment (Seteh) [inline net/DevExtreme.AspNet.Data/RemoteGrouping/RemoteGroupExpressionCompiler.cs]: Why eight?

Comment (AlekseyMartynov) [inline net/DevExtreme.AspNet.Data/RemoteGrouping/RemoteGroupExpressionCompiler.cs]: It's the number of type arguments in the `RemoteGroupKey` class. Could be increased.

**Pull request merged**